lib/modules/@apostrophecms/files-tags/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    fields: {
+      add: {
+        title: {
```

Comment:
This is correct, but is it necessary? There is already a title field configuration for all doc types by default.

Reply:
Oh ok, I thought I had to add this field. Will remove it.